code/modules/lore_codex/lore_data_vr/important_locations.dm

```diff
@@ -0,0 +1,124 @@
+	for both research and phoron refining purposes."
+
+/datum/lore/codex/page/virgo_three_bee/add_content()
+	name = "Virgo 3b (Terrestrial Moon)"
+	keywords += list("Virgo 3b")
+	data = "Virgo 3b is the only moon of [quick_link("Virgo 3")] with any atmosphere to speak of, composed primarily of [quick_link("Phoron")] and carbon dioxide with trace amounts of nitrogen. \
+	The atmosphere appears to have given rise to native life, though originally nothing bigger than small insects in terms of fauna. The flora of the planet is rather sizable, though typically blue in hue \
+	due to the atmospheric composition.\
+	<br><br>\
+	Virgo 3b is tidally locked to Virgo 3, and is currently in the 'cold phase' of Virgo 3's orbit. The 'day' consists of reflected light from the surface of Virgo 3, while the 'night' consists of a total eclipse of \
+	Virgo-Erigone by Virgo 3, leaving the planet fairly cold for a terrestrial world.\
+	<br><br>\
+	The presence of several [quick_link("TSC", "TSCs")] on the surface has introduced new life forms to the planet via bioengineering, and now several 'new' species are commonly sighted per year. \
+	Currently, [quick_link("NanoTrasen")] is constructing a 'space elevator' for commercial purposes, as the atmosphere of Virgo 3b is inimical to shuttles, and engines in particular."
+
+/datum/lore/codex/page/virgo_central_command/add_content()
+	name = "Virgo Central Command (Artificial Satellite)"
+	keywords += list("Virgo Central Command")
+	data = "The Virgo Central Command is a large facility in orbit of the planet [quick_link("Virgo 3")] which handles the loading and \
+	unloading, refuelling, and general maintenance of large spacecraft. The main structure is owned by \
+	[quick_link("NanoTrasen")], but individual offices, docking/loading bays, and warehouses are often leased to individuals \
+	or organisations.  The position of the spaceport allows it to function not only as a key node for transport inside the Virgo-Erigone \
+	system, especially to and from the planet Virgo 3, but also as a key stopping point for interstellar craft travelling via Virgo-Erigone which need refuelling. \
+	<br><br>\
+	The station itself is mostly designed around its logistical and commercial needs, and although other strategically-placed \
+	nearby facilities owned by a mixture of corporations and entities may possess habitation space, the port itself is not \
+	designed to be a living habitat - its proximity to Virgo 3 makes transport of people and materials to and from \
+	the facility and the planet's stations via shuttle extremely cost-efficient."
+
+/datum/lore/codex/page/magni/add_content()
+	name = "Virgo 4 (Terrestrial Planet)"
+	keywords += list("Magni")
+	data = "Outside of the habitable zone, the barren world Magni is generally at 202 kelvin (-71�C)."
+
+/datum/lore/codex/page/nss_adephagia/add_content()
+	name = "NSS Adephagia (Artificial Satellite)"
```

Comment:
NSB Adephagia now, since it's a base on an asteroid. NSS = NanoTrasen Science _Station_, while B = Base.